charts/pega/charts/installer/templates/_pega-environment-config.tpl

```diff
@@ -0,0 +1,59 @@
+{{- define "pega.installer.environment.config" -}}
+data:
+  # Temporary password for administrator@pega.com that is used to install Pega Platform and used to login after upgrading Pega Platform
```

Comment:
```suggestion
  # Temporary password for administrator@pega.com that is used for the first login to Pega Platform on new installations.
```
Usually only relevant to installs. Only in exceptional circumstances is it needed for an upgrade. Probably not worth calling that out here as it gives the wrong impression of usage.

Reply:
Fair enough, was trying to capture the comments from both install + upgrade configmaps.
Upgrade: `  # Temporary password for administrator@pega.com that is used to login after upgrading Pega Platform`
But if that causes more confusion than it prevents, I can update this